deployment/datasets/attributes.csv

```diff
@@ -27,10 +27,20 @@ subsection_agg,subsection_agg,,subsection_variable,string,,TRUE,FALSE,TRUE,FALSE
 id,id,,tree,int,,FALSE,TRUE,TRUE,TRUE,TRUE,TRUE,FALSE,TRUE,,,,,,,,,,,,,
 section_id,section_id,,tree,xref,section,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
 subsection_id,subsection_id,,tree,xref,sub_section,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
-id,id,,variable,int,,FALSE,TRUE,TRUE,TRUE,TRUE,TRUE,FALSE,TRUE,,,,,,,,,,,,,
-name,name,,variable,string,,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
-label,label,,variable,text,,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
+id,id,,variable,int,,FALSE,TRUE,TRUE,TRUE,FALSE,TRUE,FALSE,TRUE,,,,,,,,,,,,,
+name,name,,variable,string,,FALSE,FALSE,TRUE,FALSE,TRUE,FALSE,FALSE,TRUE,,,,,,,,,,,,,
+label,label,,variable,text,,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,TRUE,,,,,,,,,,,,,
 variants,variants,,variable,mref,variant,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
+definition-en,definition (EN),,variable,string,,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
+definition-nl,definition (NL),,variable,string,,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
+subvariable_of,subvariable of,,variable,xref,variable,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
+subvariables,subvariables,,variable,onetomany,variable,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,subvariable_of,,,,,,,,
+options,options,,variable,onetomany,variable_enum,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,variable,,,,,,,,
+id,id,,variable_enum,string,,FALSE,TRUE,FALSE,AUTO,FALSE,TRUE,FALSE,FALSE,,,,,,,,,,,,,
+variable,variable,,variable_enum,xref,variable,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
+code,enumeration code,,variable_enum,int,,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
+label-en,label (EN),,variable_enum,string,,TRUE,FALSE,TRUE,FALSE,TRUE,FALSE,FALSE,TRUE,,,,,,,,,,,,,
+label-nl,label (NL),,variable_enum,string,,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,
```

Comment:
Please add description , describing the fields and there use in the data-model / app

Reply:
I've added a description for the relations, cannot come up with anything more enlightening than the label for the rest of them